templates/admotion/afa-iframe.htm

```diff
@@ -0,0 +1,20 @@
+<html>
+<body style="margin:0px">
+<script language="JavaScript" type="text/JavaScript">
+function ADM_getArgs(){
+ var args=new Object(),query=location.search.substring(1),pairs=query.split("&");
+ for(var i=0;i<pairs.length;i++){
+  var pos=pairs[i].indexOf("=");
+  if(pos==-1)continue;
+  var argname=pairs[i].substring(0,pos),value=pairs[i].substring(pos+1);
+  args[argname]=unescape(value)
+ }
+ return args
+};
+var args=ADM_getArgs();
+for(var prop in args)
+	self[prop]=args[prop];
+document.writeln('<scr' + 'ipt type="text/javascr' + 'ipt" src="' + (encodeURIComponent(args.iq) || 'http://ia.nspmotion.com/delivery/') + '?p=' + encodeURIComponent(args.p) + '&sc=' + encodeURIComponent(args.sc) + '&r=' + (Math.round(Math.random() * 9999)) + (encodeURIComponent(args.test) ? '&test=true' : '') + '"></scr' + 'ipt>');
```

Comment:
Looks like this loads any script file passed in via the `iq` param? Don't think the `encode` protects us much.